packaging/docker/custom/Dockerfile

```diff
@@ -2,8 +2,32 @@ ARG GRAFANA_VERSION="latest"
 
 FROM grafana/grafana:${GRAFANA_VERSION}
 
+USER root
+
+ARG GF_INSTALL_IMAGE_RENDERER_PLUGIN="false"
+
+RUN if [ $GF_INSTALL_IMAGE_RENDERER_PLUGIN = "true" ]; then \
+    echo "http://dl-cdn.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories && \
+    echo "http://dl-cdn.alpinelinux.org/alpine/edge/main" >> /etc/apk/repositories && \
+    echo "http://dl-cdn.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories && \
+    apk --no-cache update && \
+    apk --no-cache  upgrade && \
+    apk add --no-cache --virtual .build-deps udev ttf-opensans chromium && \
+    rm -rf /var/cache/apk/* /tmp/* && \

```

Comment:
Is there any point in deleting /var/cache/apk/* when using `--no-cache`? Isn't the whole point of `--no-cache` [to not have an on-disk cache](https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache)?

Reply:
Good point :+1: